apps/system/style/software_button_manager/software_button_manager.css

```diff
@@ -122,5 +110,5 @@
 }
 
 #screen:not(.fullscreen-layout-app) #software-buttons-fullscreen-layout {
-  visibility: hidden;
+  visibility: visible;
```

Comment:
What's this rule for? AFAIK we don't currently use the fullscreen layout SHB anywhere.